### PR TITLE
:art: horizontal cards (in catalog and home)

### PR DESCRIPTION
**Pull request by pdt1806**
reasons why horizontal card is implemented:
- the design will look more similar to other web stores design, which will make the users feel familiar (which is a good thing)
- multiple products are visible on the screen at the same time
- each card takes less height (the current vertical card everywhere is a little bit weird, and this reason is somehow the same with reason no.1)

responsive web design thoroughly tested (min-width 350px for catalog, 320px for home)

### CODE REVIEW

**Comment by tobycm** [comment]
Next time pls open issue and wait for permit before start implementing right away lol

Btw can you include some pictures?

**Comment by pdt1806** [comment]
> Next time pls open issue and wait for permit before start implementing right away lol
> 
> Btw can you include some pictures?

1. ok ok
2. 
image
image

**Comment by tobycm** [comment]
What about desktop? Or is it phone only?

**Comment by tobycm** [comment]
About the build, it looks great. Nice job 👏 
However, a vertical version of the card flashed before changing into horizontal.
Any idea to fix?

Also, in product page, I think it's also better to use the horizontal form of the card

**Comment by pdt1806** [comment]
> About the build, it looks great. Nice job 👏 However, a vertical version of the card flashed before changing into horizontal. Any idea to fix?
> 
> Also, in product page, I think it's also better to use the horizontal form of the card

1. if (isMobile) ... **else** ... i guess (there was no else). And yes it's for mobile only.
2. I think you meant "vertical" instead of "horizontal" (since that's what I'm implementing), and I'm kind of against it because vertical card leaves a huge blank on the left and right hand side of the card. Horizontal cards use the most of the available space and as I mentioned earlier, it makes users feel familiar since most stores display their products horizontally (especially on mobile apps).

File: src/components/Card/index.tsx [hunk 1]
```diff
@@ -3,24 +3,66 @@ import { Link } from "react-router-dom";
 import pocketbase from "../../lib/database";
 import { Product } from "../../lib/database/models";
 
-export default function ProductCard(props: { product: Product; inProductPage?: boolean }) {
-  const { product } = props;
+export default function ProductCard(props: { product: Product; inProductPage?: boolean; isMobile?: boolean }) {
+  const { product, isMobile } = props;
   const productUrl = `/product/${product.id}`;
-  // const isMobile = useMediaQuery("(max-width: 36em)");
```
Comment (tobycm) [inline src/components/Card/index.tsx:9]: just delete the line lol

Comment (pdt1806) [reply src/components/Card/index.tsx:9]: just in case =)))

File: src/pages/Home/index.tsx [hunk 2]
```diff
@@ -76,42 +74,9 @@ export default function Home() {
...
-                <Title order={3} style={{ textAlign: "center" }} c="emochoice-blue">
-                  Souvenirs & Gifts Printing
-                </Title>
-              </Card.Section>
-              <Card.Section>
-                <Image src={threeCards[2]} alt="Souvenirs & Gifts Printing" maw={"100%"} h={"100%"} />
-              </Card.Section>
-            </Card>
-          </Link>
+          {["Clothing & Accessories Print", "Digital Printing", "Souvenirs & Gifts Printing"].map((name, index) => (
```
Comment (tobycm) [inline src/pages/Home/index.tsx:77]: nice

**Review by tobycm — APPROVED** [review]
Great job, best coder ever

**Pull request merged**